SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -914,6 +931,14 @@ void USpatialReceiver::HandleUnreliableRPC(Worker_ComponentUpdateOp& Op)
 		}
 	}
 
+	Schema_Object* FieldsObject = Schema_GetComponentUpdateFields(Op.update.schema_type);
```

Comment:
Hmmmm, could you do this on authority gained instead of here?

Reply:
Command is now used instead of ComponentFieldUpdate.